plugins/env/app/models/environment_variable.rb

```diff
@@ -0,0 +1,16 @@
+class EnvironmentVariable < ActiveRecord::Base
+  belongs_to :parent, polymorphic: true
+  belongs_to :deploy_group
```

Comment:
nope, you just add it to the "All" environment variable where deploy_group is nil